spl/src/metadata.rs

```diff
@@ -308,6 +328,16 @@ pub struct SetCollectionSize<'info> {
     pub system_program: AccountInfo<'info>,
 }
 
+#[derive(Accounts)]
+pub struct VerifyCollection<'info> {
+    pub payer: AccountInfo<'info>,
+    pub metadata: AccountInfo<'info>,
+    pub collection_authority: AccountInfo<'info>,
+    pub collection_mint: AccountInfo<'info>,
+    pub collection_metadata: AccountInfo<'info>,
+    pub collection_edition: AccountInfo<'info>,
```

Comment:
can we rename this to collection_master_edition

Reply:
@armaniferrante Renamed `collection_edition` into `collection_master_edition`.